composer.json

```diff
@@ -10,6 +10,7 @@
     "ext-date": "*",
     "ext-dom": "*",
     "ext-iconv": "*",
+    "ext-json": "*",
```

Comment:
If you add the extension here as required, it needs to be removed from the `"suggest"` section below.

Reply:
Thanks, done in 4165a25a86f917eb2ab8668b00a244c925691528.